app/components/ui/PeoplePicker/PeoplePickerLogic.test.js

```diff
@@ -132,7 +137,7 @@ describe('PeoplePicker', () => {
     expectSelectionLength(wrapper, 0)
   })
 
-  describe('integration with Search Box', () => {
+  describe('integration with Search Box', async () => {
```

Comment:
Why is `async` needed here?

Reply:
Left in by mistake after I removed an await inside this test. This can be removed @SemraSh

Reply:
@damnhipster thank you for this!